app/deepseg.cc

```diff
@@ -274,12 +276,21 @@ std::string resolve_path(const std::string& provided, const std::string& type) {
 	// 1. exactly what was provided
 	if (std::ifstream(provided).good())
 		return provided;
+	// to emulate PATH search behaviour (rule of least surprise), we stop here if provided has path separators
+	if (provided.find('/') != provided.npos)
+		return {};
+	// 1a. BACKSCRUB_PATH prefixes if set
```

Comment:
Having a bare `1a` feels like there's something missing. What about `1.` -> `0.` and `1a` -> `1.`?

Reply:
I thought I renumbered them.. will check!

Reply:
Yes, they are renumbered :smile: